### DISCUSSION TITLE
What is the Login and Password to install Elementary OS???

### DISCUSSION THREAD

**Discussion by theofficialitbros** [elementary/triage#480] [Q&A]
I have literally scoured the Internet searching for the answer to this question not been able to find it. 

I have been trying to install elementary OS locally on my machine for a week now and whether I use safe mode or does not matter I cannot get past this elementary OS Horus login and password screen and it was not provided to me when I download in the ISO from the website, so what the hell?

**Comment by danirabbit** [comment]
This is not an appropriate way to ask for help in our community. Consider this a warning.

You do not need a password to try or install elementary OS. If you're seeing a login prompt, something has gone wrong.

There's an answer [here](https://github.com/elementary/os/discussions/427) describing what it normally looks like to boot from the installation media.

There's a post [here](https://github.com/orgs/elementary/discussions/329) where someone else encountered an issue like this and it was resolved by re-downloading.

**Reply by theofficialitbros** [reply]
My apologies. This has just been very frustrating. I've actually downloaded version 7 several times from the elementary OS website and created a bootable flash drive multiple times and every single time I have come up against this issue of it booting to a screen that asks me for an elementary os horace tty1 login and password. 

I have ernabled Intel SGX And done everything that I need to do as far as my bios goes in allowing "non-windows" OS's, and yet I still can't get passed this screen..

**Reply by theofficialitbros** [reply]
https://github.com/elementary/triage/assets/150261875/69782d40-19d2-4e66-91cb-d9299e86442c

**Reply by jlnr** [reply]
elementary OS does not ship the proprietary Nvidia drivers in their installer/live ISO, only the free Nouveau driver. I guess Nouveau does not support your RTX 4090 and that's why the live session doesn't work?

I also have an Nvidia card and I keep distrohopping in circles because, even if the original installation works, something always breaks sooner or later. I hope things will get better with the new NVK driver (Nouveau's successor).

**Comment by theofficialitbros** [comment]
These are my hardware specifics:
**Processor:** Intel(R) Core(TM) i7-10700F CPU @ 2.90GHz   2.90 GHz
**Installed RAM:** 64.0 GB (63.9 GB usable)
**System type:** 64-bit operating system, x64-based processor
**Graphics Card:** MSI GeForce RTX 4090 24GB GDRR6X
